### DISCUSSION TITLE
Increase Daily Notional Limit For Polygon and Arbitrum

### DISCUSSION THREAD

**Discussion by mizdler** [wormhole-foundation/wormhole#3724] [Governance]
Increase daily notional limits for Arbitrum and Polygon to 10M.

**Comment by claudijd** [comment]
Here's some supporting data on Polygon flows to allow Guardians to make informed decisions here:

**Polygon: 30 Day**

Wormhole chain id 5 token bridge transfers over the past 31 days between sequences 188668 and 188491:

Notional usd average : $1,716,205.57
Notional usd total   : $53,202,372.74

**Polygon: 6 Month**

Wormhole chain id 5 token bridge transfers over the past 181 days between sequences 188541 and 168835:

Notional usd average : $401,135.86
Notional usd total   : $72,605,589.80

The Polygon current limit is set to $2M and current Governor usage over the past day is 82% of the capacity.  Adjusting the Polygon governor limit to $10M as proposed would represent 5x the current usage.  It would seem to me that $5M might be a more conservative step up for Guardians to consider, allowing for 3x headroom on recent usage, and based on recent flows Polygon demand appears to be on the upswing.

I'll do a similar analysis on Arbitrum and provide the output shortly.

**Comment by SEJeff** [comment]
Polygon was upped in #3734

**Comment by SEJeff** [comment]
Arbitrum stats over the past 30 days:
```bash
Wormhole chain id 23 token bridge transfers over the past 31 days between sequences 72161 and 147477:

Notional usd average : $3,888,751.81
Notional usd total   : $120,551,306.08
```

Arbitrum stats over the past ~6 months:
```bash
Wormhole chain id 23 token bridge transfers over the past 181 days between sequences 8933 and 119091:

Notional usd average : $1,212,976.06
Notional usd total   : $219,548,666.70
```

The current Arbitrum limit is at $5 million per: https://github.com/wormhole-foundation/wormhole/blob/3f074f30eed3cd94172a1a24cbd7adb1985bb70d/node/pkg/governor/mainnet_chains.go#L33 so the average daily max over the past month is 76% (on average) of the limit.

**Comment by SimonKraus** [comment]
Thanks a lot @SEJeff for the additional information.

On one hand we just highered the limits less than a month ago - but on the other hand hitting 76% of the daily limits in the past 30 days is an indicator of rapid growth and the need of another increase for the governor limits. 

That would give us some room to breath. 

Staking Facilities supports the 🆙
